Packs/CloudIncidentResponse/Playbooks/playbook-Cortex_XDR_-_Cloud_Data_Exfiltration_Response.yml

```diff
@@ -1047,19 +1048,176 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "86":
+    id: "86"
+    taskid: b4d64aed-f0ba-4f76-83a6-67e469740ba7
+    type: playbook
+    task:
+      id: b4d64aed-f0ba-4f76-83a6-67e469740ba7
+      version: -1
+      name: Cloud Credentials Rotation - Generic
```

Comment:
In the other playbook, you added a conditional task whether to run this subplaybook. why don't we have it here?